backend/src/services/authService.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require('jsonwebtoken');
+
+export const authService = (token,secret) => {
```

Comment:
@gulyaasjaanos the secret can be here a constant in the service (read from env), it's not really going to be changed anywhere

Reply:
fully agreed, I'll make that changes

Reply:
done